include/gmock-win32.h

```diff
@@ -55,8 +55,16 @@ struct mock_module_##func \
 #define MOCK_MODULE_FUNC0_CDECL_CONV(m, ...) MOCK_MODULE_FUNC0_CALLCONV(__cdecl, m, __VA_ARGS__)
 
 #define MOCK_MODULE_FUNC1_(tn, constness, ct, func, ...) \
+_Pragma( "optimize( \"\", off )" ) \
 struct mock_module_##func \
 { \
+    static void hideCodeFromOptimizer() { \
```

Comment:
How about renaming the function from `hideCodeFromOptimizer` to something like `patchModuleFunc` or another suitable name?

Reply:
Of course I'll do it.